serenity-screenplay-webdriver/src/test/java/net/serenitybdd/screenplay/webtests/pages/JLHomePage.java

```diff
@@ -0,0 +1,42 @@
+package net.serenitybdd.screenplay.webtests.pages;
+
+import net.serenitybdd.core.annotations.findby.By;
+import net.serenitybdd.core.annotations.findby.FindBy;
+import net.serenitybdd.core.pages.PageObject;
+import net.serenitybdd.core.pages.WebElementFacade;
+import net.serenitybdd.screenplay.targets.Target;
+import net.serenitybdd.screenplay.waits.WaitUntil;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import static net.serenitybdd.screenplay.matchers.WebElementStateMatchers.isCurrentlyVisible;
+
+import java.time.Duration;
+
+public class JLHomePage extends PageObject {
+    private static final String SITE = "#siteMenu";
+    private static final String ADDSITE = "#createSiteMenu";
+
+    public static Target HRFSITE = Target.the("SiteHref").locatedBy(SITE);
+    public static Target MNUADDSITE = Target.the("AddSiteMenuItem").locatedBy(ADDSITE);
+
+    @FindBy(id = "siteMenu")
+    public WebElementFacade site;
+
+    @FindBy(id = "createSiteMenu")
+    public WebElementFacade addSite;
+
+    public void navigateToPage(String pageName) {
```

Comment:
If you are doing a screenplay implementation, the navigation logic should be a Task or Interaction, not a Page Object method. In Screenplay, Page Objects are strictly limited to containing locators and locator logic.

Reply:
Hi @wakaleo I'm sorry for my mistakes to create this pull request by accidentally. However, I'll appreciate for your kind feedback. Could you please me to close this one? Appreciate and thank you!